Cargo.lock

```diff
@@ -6030,12 +6206,13 @@ dependencies = [
  "regex",
  "regex-syntax 0.7.2",
  "reqwest",
- "ring",
+ "ring 0.16.20",
  "rustls",
  "scopeguard",
  "serde",
  "serde_json",
  "smallvec",
+ "subtle",
```

Comment:
TODO(me): check what our duplicate deps look like now since this shuffles a lot of those.

Reply:
Impressive: aws-sigv4 brings two versions of crypto-bigint.

Other than it looks like our only old ring dependency is jsonwebtoken which has gone through a major bump, possibly for ring... I think that's all new duplicates.

Reply:
@koivunej, Are we going to fix it in this PR?

Reply:
No no, this was just for myself (and internally for anyone interested in duplicate deps and why we cannot update them).